doc/changelogs/CHANGELOG_V8.md

```diff
@@ -2474,7 +2474,7 @@ Big thanks to @addaleax who prepared the vast majority of this release.
 * [[`16f2600ecf`](https://github.com/nodejs/node/commit/16f2600ecf)] - **child_process**: emit IPC messages on next tick (cjihrig) [#13856](https://github.com/nodejs/node/pull/13856)
 * [[`dfc46e262a`](https://github.com/nodejs/node/commit/dfc46e262a)] - **(SEMVER-MINOR)** **cluster**: overriding inspector port (cornholio) [#14140](https://github.com/nodejs/node/pull/14140)
 * [[`26f85e75f9`](https://github.com/nodejs/node/commit/26f85e75f9)] - **cluster**: remove obsolete todo (Ruben Bridgewater) [#13734](https://github.com/nodejs/node/pull/13734)
-* [[`816f98f5d0`](https://github.com/nodejs/node/commit/816f98f5d0)] - **console**: use a plain object for the the error stack (Ruben Bridgewater) [#13743](https://github.com/nodejs/node/pull/13743)
```

Comment:
`13743)` -> `1374` is not required, might have been added by mistake.

Reply:
Thanks. Fixed it.

Reply:
This should not be fixed. The commit message itself is wrong and therefore it should stay the way it is.